packages/template-retail-react-app/app/pages/checkout/index.test.js

```diff
@@ -588,135 +580,4 @@ test.skip('Can edit address during checkout as a registered customer', async ()
     await waitFor(() =>
         expect(screen.getByTestId('sf-toggle-card-step-1-content')).not.toBeEmptyDOMElement()
     )
-
-    const firstAddress = screen.getByTestId('sf-checkout-shipping-address-0')
-    user.click(within(firstAddress).getByRole('button', {name: /edit/i}))
-
-    // Wait for the edit address form to render
-    await waitFor(() =>
-        expect(screen.getByTestId('sf-shipping-address-edit-form')).not.toBeEmptyDOMElement()
-    )
-
-    expect(screen.getByRole('textbox', {name: /first name/i})).toBeInTheDocument()
-
-    // Edit and save the address
-    user.clear(screen.getByLabelText('Address'))
-    user.type(screen.getByLabelText('Address'), '369 Main Street')
-    user.click(screen.getByText(/save & continue to shipping method/i))
-
-    // Wait for next step to render
-    await waitFor(() => {
-        expect(screen.getByTestId('sf-toggle-card-step-2-content')).not.toBeEmptyDOMElement()
-    })
-
-    expect(screen.getByText('369 Main Street')).toBeInTheDocument()
-})
-
-// test('Can add address during checkout as a registered customer', async () => {
```

Comment:
It may not be clear... but this test was commented out, but I've brought it back.